saleor/graphql/order/types.py

```diff
@@ -1178,6 +1183,7 @@ class Order(ModelObjectType[models.Order]):
         deprecation_reason=(f"{DEPRECATED_IN_3X_FIELD} Use `id` instead."),
     )
     voucher = graphene.Field(Voucher)
+    voucher_code = graphene.Field(VoucherCode)
```

Comment:
Also missing description and `ADDED_IN_X` label

Reply:
added